modules/web-console/web-agent/README.txt

```diff
@@ -32,6 +32,7 @@ Configuration file:
     server-trust-store
     server-trust-store-password
     passwords-key-store
+    passwords-key-store-password
```

Comment:
We don't support the passwords-key-store-password in config, only user prompt or command line argument.

Reply:
Fixed.